### ISSUE TITLE
Drag items from one list to another

### ISSUE THREAD

**Issue by perqa** [computerjazz/react-native-draggable-flatlist#503]
**Is your feature request related to a problem? Please describe.**
In a nested list setup, is it possible to drag items from one nested list to another?

**Describe the solution you'd like**
A configuration parameter to allow or disallow dragging items between lists. Limited to lists on the same level, in case of nesting.

**Describe alternatives you've considered**
Forking and modifying the library, but I was hoping there is a better way to do it.

**Comment by AndreiSager** [comment]
I am also curious. Can I use this to create a kanban board?

**Comment by codeandcoffee-io** [comment]
Did either of you ever get an answer or implement a solution? :)

**Comment by AndreiSager** [comment]
@codeandcoffee-io no, I just used a different package. Here it is https://github.com/VolkenoMakers/react-native-drag-drop. It might not be the best but I made it work somehow for my app.

**Comment by codeandcoffee-io** [comment]
@AndreiSager Thanks for the feedback! I ended up just altering my design tbh. I saw that library and it gave me red flags with no commits in the last 2 years haha!
